aiven/client/cli.py

```diff
@@ -138,10 +141,10 @@ def get_current_date() -> datetime:
 
 
 if (sys.version_info.major, sys.version_info.minor) >= (3, 8):
-    from typing import Protocol  # pylint: disable=no-name-in-module
+    from typing import Protocol
 
-    class ClientFactory(Protocol):  # pylint: disable=too-few-public-methods
-        def __call__(self, base_url: str, show_http: bool, request_timeout: Optional[int]) -> client.AivenClient:
+    class ClientFactory(Protocol):
+        def __call__(self, base_url: str, show_http: bool, request_timeout: int | None) -> client.AivenClient:
```

Comment:
nit: we can remove the (now redundant)`if` guard here which is for 3.8. I thought `pyupgrade` took care of this pattern, maybe `ruff`'s pyupgrade rules don't ?